mmdet/datasets/pipelines/loading.py

```diff
@@ -100,28 +102,23 @@ class LoadMultiImagesFromMultiFiles(object):
     """
 
     def __init__(self,
-                 img_info_prefix_keys=['target', 'ref'],
+                 prefixs=['target', 'ref'],
                  to_float32=False,
                  color_type='color',
                  file_client_args=dict(backend='disk')):
-        self.img_info_prefix_keys = img_info_prefix_keys.copy()
-        self.img_info_keys = [
-            img_info_prefix_key + '_imgs_info'
-            for img_info_prefix_key in self.img_info_prefix_keys
-        ]
+        self.prefixs = prefixs.copy()
```

Comment:
Is `copy()` necessary here?

Reply:
Mutable variables should be copied when using them as arguments.